src/Auth/HasIdentity.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Tempest\Auth;
+
+use Tempest\Database\IsModel;
+
+trait HasIdentity
+{
+    use IsModel;
+
+    protected const string IDENTIFIER = 'email';
```

Comment:
I don't think this is the way to go, as these can't be enforced with the interface. Instead provide two getters for it in the interface, and implement default implementations in the trait.

`getIdentifierFieldName` `getSecretFieldName`, something like that

Reply:
We have the method, I’ll remove the constants